src/Raven.Server/Web/System/Processors/Backups/BackupDatabaseHandlerProcessorForGetPeriodicBackupStatus.cs

```diff
@@ -61,7 +61,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandl
 
             foreach (var status in statuses)
             {
-                status.Dispose();
+                status?.Dispose();
```

Comment:
side quest: let's change `Statuses` to be a dictionary, since now we can have not consecutive shard numbers and we want to have the info which status is for which shard

Reply:
cc @ml054

Reply:
done